src/configure/clients/devOps/azureDevOpsClient.ts

```diff
@@ -226,19 +232,19 @@ export class AzureDevOpsClient {
             serializationMapper: null,
             deserializationMapper: null
         })
-        .then((project) => {
-            return project && project.id;
-        });
+            .then((project) => {
```

Comment:
here and other places as well. please do not change indent.

Reply:
This got changed with auto formatting setting from vscode. Keeping this change for now, will remove the setting from editor so it doesn't happen again.